index.rst

```diff
@@ -9,6 +9,19 @@ User Documentation
    accountsettings/index.rst
    workbench/workbench_index.rst
    maintenance/maintenance_index.rst
+   cookbook/whitelabel
+   cookbook/jstranslation
+   cookbook/resourcetranslation
+   cookbook/hubspotforms
+   cookbook/pageviewsandcaching
+   cookbook/requestevaluation
+   cookbook/stagingdomain
+   cookbook/rtlconversion
+   cookbook/ssl_certificates
+   cookbook/proxymodes
+   cookbook/headers
+   cookbook/securelogin
+   cookbook/sitesearch
```

Comment:
Should these be given a separate section in the index? The way they are now, they kinda blend together with the rest of the content...